app/org/maproulette/framework/mixins/SearchParametersMixin.scala

```diff
@@ -102,7 +102,7 @@ trait SearchParametersMixin {
       List(
         FilterParameter.conditional(
           Project.FIELD_DISPLAY_NAME,
-          s"'${SQLUtils.search(params.projectSearch.getOrElse("").replace("'", "''"))}'",
+          s"'${SQLUtils.search(params.projectSearch.getOrElse(""))}'",
```

Comment:
Why was this removed? Is it executed somewhere else? If you don't remove single quotes it will fail in the database.

Reply:
Reading your description, it seems that it would indicate that you should leave it in. I might be misreading it though.